matlab/outputFileConflictCheck.m

```diff
@@ -0,0 +1,14 @@
+function result = outputFileConflictCheck (directory, file_name)
+    [~,name,ext] = fileparts(file_name);
+    output_file_name = [name '_output' ext];
+
+    for file = dir(directory)'
```

Comment:
I don’t know Matlab so sorry if this is a stupid question—what is the purpose of that apostrophe at the end of the line?

Reply:
@jtorre39 answered verbally: the apostrophe stands for matrix transposition 😳